com.unity.formats.alembic/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 ### Chnages
 - Updated to Alembic version 1.7.16.
 - Added support for Stadia standalone builds.
+- Add support for Arm64 OSX.
```

Comment:
I didn't really find an official way to properly write down the name of the thing but here is my suggestion after putting together some research results:
`- Added support for arm64 macOS.`

Reply:
Fixed